wiki/Performance_points/en.md

```diff
@@ -26,7 +24,7 @@ On January 16, 2021, changes were made to the ppv2 system that aimed to more acc
 - Adjust the rate of pp loss from misses to be more forgiving on longer maps with higher combo
 - Punish lower [accuracy](/wiki/Gameplay/Accuracy) plays with less pp gain
 
-ppv2 is currently in active service, with live updates published to its [changelog](https://osu.ppy.sh/p/changelog?category=pp).
+ppv2 is currently in active service, with live updates published by the [PP Committee](/wiki/en/People/Performance_Points_Committee)
```

Comment:
```suggestion
ppv2 is currently in active service, with live updates published by the [PP Committee](/wiki/People/Performance_Points_Committee)
```

Reply:
sorry I almost missed this suggestion lol